tests/saw/spec/handshake/handshake_io_lowlevel.saw

```diff
@@ -382,15 +384,13 @@ let s2n_advance_message_spec = do {
     // in-memory representation of the arrays hasn't changed, just the types.
     // As a result, we can use crucible_points_to_untyped to check if the two
     // state machines are bitwise equivalent without checking the types.
-    // TLS1.2 is temporarily removed from SAW testing
-    // Tracking issue: https://github.com/aws/s2n-tls/issues/3516
-    // crucible_points_to_untyped (crucible_global "handshakes") (crucible_term {{ handshakes }});
+    crucible_points_to_untyped (crucible_global "handshakes") (crucible_term {{ handshakes }});
     crucible_points_to_untyped (crucible_global "tls13_handshakes") (crucible_term {{ tls13_handshakes }});
     
     let messages = [ {{CLIENT_HELLO : [5]}}
                    , {{SERVER_HELLO : [5]}}
-                   , {{SERVER_CERT : [5]}}
                    , {{SERVER_NEW_SESSION_TICKET : [5]}}
+                   , {{SERVER_CERT : [5]}}
```

Comment:
Why was the order changed? Just curious, since I can't figure out if this change affects anything.